docs/docs/01-start-here/03-hello.md

```diff
@@ -26,7 +26,7 @@ bring cloud;
 let bucket = new cloud.Bucket();
 let queue = new cloud.Queue();
 
-queue.setConsumer(inflight (message: str) => {
+queue.addConsumer(inflight (message: str) => {
```

Comment:
The method is called `setConsumer`, not `addConsumer`... am I missing something out?
[Docs](https://www.winglang.io/docs/standard-library/cloud/queue#@winglang/sdk.cloud.Queue.setConsumer)

Reply:
My bad, old wing version, thought the docs were outdated :)
Closing

Reply:
Thanks!